code/player/Player.cs

```diff
@@ -98,6 +103,26 @@ public override void Simulate(Client client)
             TickPlayerUse();
             TickAttemptInspectPlayerCorpse();
 
+            if (Input.Pressed(InputButton.Drop) && ActiveChild != null && Inventory != null)
+            {
+                int weaponSlot = (int) (ActiveChild as Weapon).WeaponType;
+                Entity droppedEntity = Inventory.DropActive();
+
+                if (droppedEntity != null)
+                {
+                    if (droppedEntity.PhysicsGroup != null)
+                    {
+                        droppedEntity.PhysicsGroup.Velocity = Velocity + (EyeRot.Forward + EyeRot.Up) * 300;
```

Comment:
Let's save this `300` magic number as a static variable in this class. We can probably make a ttt server param for it as well.